src/Caller.h

```diff
@@ -123,7 +128,79 @@ class Caller {
   virtual void processPeptideScores(Scores& allScores) {}
   virtual void processProteinScores(ProteinProbEstimator* protEstimator) {}
 #endif
+
+bool detect_tab(std::string file_name)
```

Comment:
Place this as a (static?) method of a Class. Possibly TabReader.

Reply:
The function has been moved DataSet and changed name here: 05b0a8a. But made static here: https://github.com/percolator/percolator/commit/f82b2b1320ed98ac2ec95f33a3c9bf6e3d7a2d9c